### ISSUE TITLE
Did 0.4.0 break testing local starter kyts?

### ISSUE THREAD

**Issue by janhoogeveen** [NYTimes/kyt#401]
### Versions of kyt, node, OS
kyt 0.4.0 after following the migration guide

### Bug or Feature?
Bug

### Current Behavior
in 0.3.0 I was able to use the bootstrap command and test a local starter-kyt, this made sure my Github only shows clean commits. I updated to kyt 0.4.0 and want to test my changes before pushing them to Github. As I understand it the transition to a monorepo with kyt-cli means we don't need to install the monorepo and use bootstrap commands anymore to perform command line magic.

These are the 2 scenario's I've tried.
First, installing it with a globally installed kyt-cli

```
✔ 17:24 ~/Projects/cleverfranke $ kyt-cli setup -d test -r kyt/.git --repository-path ./cf-kyt-starter-universal-redux/
0.19.1

🔥  Let's set up your new kyt project...

✨  Answer a few questions to get started  ✨

? Choose an installer yarn
👍  Creating your new project at test
👍  Setting up the specified starter-kyt
fatal: repository 'kyt/.git' does not exist


❌  There was a problem cloning the repository

fatal: repository 'kyt/.git' does not exist


❌  Failed to setup: kyt/.git
```

Second try, a locally installed kyt-cli

```
✔ 17:35 ~/Projects/cleverfranke/cf-kyt-test [master L|…2261] $ node_modules/.bin/kyt-cli setup -r kyt-cli/.git --repository-path ../cf-kyt-starter-universal-redux
0.19.1

🔥  Let's set up your new kyt project...

✨  Answer a few questions to get started  ✨

? Choose an installer yarn
? Enter a new directory name. To install in current directory, leave blank.
👍  Setting up the specified starter-kyt
fatal: repository 'kyt-cli/.git' does not exist


❌  There was a problem cloning the repository

fatal: repository 'kyt-cli/.git' does not exist


❌  Failed to setup: kyt-cli/.git
```

### Desired Behavior
After installing kyt-cli (globally or locally) I want to be able to pinpoint a local directory and use that as a starter kyt.

**Comment by janhoogeveen** [comment]
Worth mentioning is the fact that I would be fine with specifying a specific branch/commit/tag as well when using the kyt-cli setup command and just setup from a public repo. For example, this would be a-okay in my book:

```
kyt-cli setup -r https://github.com/cleverfranke/cf-kyt-starter-universal-redux.git#feature/migrate-0.4.0
```

**Comment by ccpricenytimes** [comment]
Hey @janhoogeveen this should be fixed in the latest version of kyt-cli (0.2.0) You can use --local-path to pass in your local starter-kyt

https://github.com/NYTimes/kyt/blob/master/CONTRIBUTING.md#testing-local-starter-kyt-changes

**Comment by janhoogeveen** [comment]
Works like a charm, thanks so much!
